ivy/src/main/scala/sbt/internal/librarymanagement/Ivy.scala

```diff
@@ -313,7 +313,7 @@ final class IvySbt(
         mod.setHomePage(h.toString)
       }
       moduleInfo.licenses foreach { l =>
-        mod.addLicense(new License(l._1, l._2.toString))
+        mod.addLicense(new License(l.name, l.url.toString))
```

Comment:
As can be seen here, the Ivy API hardcoded license to only contain 2 fields however we can also add the extra `distribution`/`comments` field to the raw POM manifest itself since it is part of maven.

I haven't done this currently but if people agree with the direction of the PR I can add this functionality in there.